Dockerfile

```diff
@@ -3,6 +3,4 @@
 # https://github.com/docker-library/repo-info/tree/master/repos/alpine/remote
 FROM alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad
 ARG EXTENSION_FILE
-COPY ${EXTENSION_FILE} /opt/elastic-apm-extension
```

Comment:
This might cause some issues if `EXTENSION_FILE` != `elastic-apm-extension`, I don't know if `EXTENSION_FILE` will be always `elastic-apm-extension` but since it's an ARG I might say this line will be needed

Reply:
https://github.com/elastic/apm-aws-lambda/pull/362/commits/38753055563a324cf0231c74c3d80506e311e30a